### PR TITLE
Change route prefixes in security.yaml

### PR DESCRIPTION
**Pull request by jacquesbh**
| Q               | A
| --------------- | -----
| Branch?         | 1.8
| Bug fix?        | no, not really
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets |
| License         | MIT

Since we had the same change in the `access_control`, I've done it in the firewalls as well.

😘

_I'm not sure about the UPGRADE-1.8.md actually. Changing the version number is maybe not the best thing to do._

### CODE REVIEW

**Review by lchrusciel — APPROVED** [review]

File: UPGRADE-1.8.md [hunk 1]
```diff
@@ -1,10 +1,19 @@
...
+             firewalls:
+                 new_api_admin_user:
+                     json_login:
+    -                    check_path: "%sylius.security.new_api_route%/admin/authentication-token"
+    +                    check_path: "%sylius.security.new_api_admin_route%/authentication-token"
+                 new_api_shop_user:
+                     json_login:
+    -                    check_path: "%sylius.security.new_api_route%/shop/authentication-token"
+    +                    check_path: "%sylius.security.new_api_shop_route%/authentication-token"
+             access_control:
```
Comment (Zales0123) [inline UPGRADE-1.8.md:16]: Maybe it would be better to introduce it in `1.8.5` -> `1.8.6` upgrade note? But I don't know isn't it too finicky 💃

**Review by Zales0123 — APPROVED** [review]

File: UPGRADE-1.8.md (hunk 1 above)
Comment (jacquesbh) [reply UPGRADE-1.8.md:16]: Yeah, I thought about it… and had the same hesitation 😅

Comment (jacquesbh) [reply UPGRADE-1.8.md:16]: This should be in 1.8.5 actually… it's a mistake not to have this change in the previous PR :p.

That's why I hesitate. Let me know about it and I'll do the change if needed. It's not complicated :p

Comment (jacquesbh) [reply UPGRADE-1.8.md:16]: @Zales0123
Do I change? Or not? What do you want me to do?

**Comment by Zales0123** [comment]
Thanks, Jacques! :1st_place_medal:

File: UPGRADE-1.8.md (hunk 1 above)
Comment (Zales0123) [reply UPGRADE-1.8.md:16]: Sorry for missing this comment, let it be as it is 😄

**Pull request merged**